editor/plugins/spatial_editor_plugin.h

```diff
@@ -327,6 +327,7 @@ class SpatialEditorViewport : public Control {
 	Vector<_RayResult> selection_results;
 	bool clicked_includes_current;
 	bool clicked_wants_append;
+	bool selection_in_progress = false; 
```

Comment:
There's an extra space at the end of the line.

Reply:
Fixed